modules/tinymce/src/themes/silver/main/ts/ui/context/ContextToolbarLookup.ts

```diff
@@ -5,19 +5,63 @@
  * For commercial licenses see https://www.tiny.cloud/
  */
 
-import { Toolbar } from '@ephox/bridge';
 import { Node as DomNode } from '@ephox/dom-globals';
 import { Arr, Option } from '@ephox/katamari';
 import { Compare, Element, TransformFind } from '@ephox/sugar';
 import Editor from 'tinymce/core/api/Editor';
-
+import { ContextTypes } from '../../ContextToolbar';
 import { ScopedToolbars } from './ContextToolbarScopes';
 
-export type LookupResult = { toolbarApi: Toolbar.ContextToolbar | Toolbar.ContextForm, elem: Element };
+export type LookupResult = { toolbars: Array<ContextTypes>, elem: Element };
```

Comment:
These shouldn't use `Array` as it's not a complex type, so it should just be `ContextTypes[]`. We really need to fix that tslint rule for TinyMCE, as it works for alloy but not TinyMCE :thinking:

Reply:
Hmm, actually maybe it's alloy that's configured incorrectly? As the tslint-rules default is to use `"array-type": false`. Probably worth discussing what we should be using.

Reply:
I really don't know when we're meant to use which tbh. Tslint rules would be nice.

Reply:
`X[]` and `Array<X>` are equivalent, so it really is just a style choice